### PR TITLE
added Source for Korneuburg [Austria]

### PR DESCRIPTION
**Pull request by 53RT**
Added support Korneuburg in Austria

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        entries = [Collection(d[0], d[1]) for d in dates]
+
+        return entries
+
+    def fetch(self):
+
+        ical_urls = self.get_region_links()
+
+        for u in ical_urls:
+            print(u)
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py:173]: Remove debug output

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    "Rathaus using Teilgebiet": {"street_name": "Some Street", "street_number": "1A", "teilgebiet": "4"},  # Teilgebiet 4
+    "Werft": {"street_name": "Am Hafen", "street_number": 6}                                               # Teilgebiet 2
+}
+
+
+def http_get_retry(params, n_retries=5):
+    """HTTP GET with n retries """
+    s = requests.Session()
+
+    retries = Retry(total=n_retries,
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py:33]: Just to be curious: Is their service so unreliable that a special retry handling is required? I have never seen a need for this in any other source.

**Review by mampfes — CHANGES_REQUESTED** [review]

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py (hunk 1 above)
Comment (53RT) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py]: Thanks. I have overlooked this one.

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py (hunk 2 above)
Comment (53RT) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py:33]: When I was writing the code I often got 502 responses and this helped so I could run all the tests. 

Today the tests ran quite fast so I think it was a temporary issue. If you prefer I can remove this function.

Comment (mampfes) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py:33]: Yes, I would prefer to start without this special code. Please remove it. Thanks.

Comment (53RT) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/korneuburg_stadtservice_at.py]: Okay. Done

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**